src/main.cpp

```diff
@@ -653,6 +653,9 @@ int main(int argc, char *argv[]) {
 #ifdef Q_OS_MAC
         loadMacTranslations(translatorOSX, translatorOSX2, translatorOSX3, translatorOSX4,
                             QCoreApplication::applicationDirPath(), locale);
+
+    // disable icons in the menu
+    QApplication::instance()->setAttribute(Qt::AA_DontShowIconsInMenus, true);
```

Comment:
Can you please use the same intend like in the lines above. 😅

Reply:
done.